util/install

```diff
@@ -73,9 +78,10 @@ check_music_tools()
   fi
 
   CDPARANOIA=`which cdparanoia`
+  FLAC=`which flac`
```

Comment:
Let's focus on one change at a time - device parameter first, then we can discuss flac.
(or you could issue a separate PR from a separate branch for the parallel flac discussion)